utils/select-fields.js

```diff
@@ -70,16 +71,14 @@ function expand(object, path, paths) {
 
 function createSanitizerMapper(req, { sanitizers = [] }, { sanitize = true }) {
   return !sanitize ? field => field : field => sanitizers.reduce((prev, sanitizer) => {
-    const value = typeof prev.value === 'string' ?
-      callSanitizer(sanitizer, prev) :
-      prev.value;
-
+    const value = !!prev.value ? callSanitizer(sanitizer, prev) : prev.value;
```

Comment:
Not sure why I needed to a this check. If ended up just calling callSanitizer(sanitizer, prev) test would fail. Mainly because the prev.value is undefined.